templates/frontend/pages/sections.tpl

```diff
@@ -0,0 +1,73 @@
+{**
+ * templates/frontend/pages/sections.tpl
```

Comment:
This should be `sections.tpl` rather than `sections.tpl` because it shows articles in one section.